### DISCUSSION TITLE
Make atmos easier to find

### DISCUSSION THREAD

**Discussion by hiitachi** [tidal-music/discussions#58] [Ideas & Feature requests]
On nvida shield app, or any app I think, can you please put the atmos albums available with the rest of the albums on the artists page..Or add a search/filter feature to specifically show atmos albums available.

 It's super annoying having to do a general search for an album, and then cycle through 20 albums searched before hitting the atmos album burried in the search results every time. 

 It feels like the atmos albums are always a buried feature when searching,  and that is just a bad experience

**Comment by stefanmoro** [comment]
Hey @hiitachi ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API, open source projects etc).

As this question relates to an existing product feature, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
